README.md

```diff
@@ -23,89 +23,18 @@ For autotrack, add the following plugins to your `.babelrc` or `babel.config.js`
-
-```ruby
-pod "react-native-heap", path: "../node_modules/@heap/react-native-heap"
-```
-
-Then run:
-
-```bash
-pod install
-```
-
-You're done! :tada:
-
-### Manual setup - iOS without Cocoapods
-
-All terminal commands assume you are in the top-level directory of your React Native project.
-
-1. Locate the Xcode project for RNHeap: `open node_modules/@heap/react-native-heap/ios` . You should see a Finder window containing the `RNHeap.xcodeproj` file.
-1. Open the Xcode project for your native app, in the `ios` directory of your project.
-1. Drag the `RNHeap.xcodeproj` file from the Finder window into the "Libraries" group of your Xcode project.
-1. Select your project name in the Xcode project navigator. Select the "Build Phases" tab for your app's target.
-1. Expand the build phase called "Link Binary With Libraries". In the project navigator on the left, expand the `RNHeap.xcodeproj` entry, and find `libRNHeap.a` in the "Products" group. The icon for `libRNHeap.a` should look like a little building with columns.
-1. Drag the `libRNHeap.a` file into the list of other libraries to be linked.
-1. Expand the build phase called "Copy Bundle Resources". In the project navigator on the left, expand the `RNHeap.xcodeproj` entry, and find `HeapSettings.bundle`. Drag this into the list of bundle resources to be copied.
-
-**NOTE**: Using `react-native link` is not currently supported
-
-### Manual setup - Android
-
-In some cases, simply doing `react-native link` will correctly set up the tracker on Android.
-However, if you encounter any issues (either with the command itself or at compile-time/run-time),
-you should revert any changes made by `react-native link` and try the following steps, instead.
-
-Add the following to your `settings.gradle`:
-
-```groovy
-include ':react-native-heap'
-project(':react-native-heap').projectDir =
-    new File(rootProject.projectDir, '../node_modules/@heap/react-native-heap/android')
-```
-
-Then add the following to `app/build.gradle`:
-
-```groovy
-implementation project(':react-native-heap')
-```
-
-Import the package and add it to your app's MainApplication:
-
-```java
-
-@Override
-import com.heapanalytics.reactnative.RNHeapLibraryPackage;
-...
-protected List<ReactPackage> getPackages() {
-  return Arrays.<ReactPackage>asList(
-    new MainReactPackage(),
-    new RNHeapLibraryPackage()
-  );
-}
-```
-
-If you are seeing runtime warnings at startup mentioning `Heap: Could not find BuildConfig`, add the
-following line to the resources section of `res/values/strings.xml`:
-
-```xml
-<string name="com.heapanalytics.android.buildConfigPkgName">com.your_package_name</string>
-```
-
-and replace `com.your_package_name` with the package name from the manifest tag in `AndroidManifest.xml`.
+- [iOS](docs/ios-setup.md)
+- [Android](docs/android-setup.md)
 
 ## Usage
 
 ```js
 // Import Heap.
 import Heap from '@heap/react-native-heap';
 
-// Start Heap.
+// Start Heap if not using auto-initialization.
```

Comment:
If we expect auto-init to be the common case let's move this line out of the usage entirely and hide it away in the "manual initialization" section.